src/operator_ceed.c

```diff
@@ -481,7 +605,11 @@ PetscErrorCode CreateCeedFluxOperator(RDyConfig *config, RDyMesh *mesh, PetscInt
   // flux suboperator 0: fluxes between interior cells
 
   CeedOperator interior_flux_op;
-  PetscCall(CreateCeedInteriorFluxOperator(*config, mesh, &interior_flux_op));
+  if (config->numerics.flux_single_comm) {
+    PetscCall(CreateCeedAllInteriorFluxOperator(*config, mesh, &interior_flux_op));
+  } else {
+    PetscCall(CreateCeedInteriorFluxOperator(*config, mesh, &interior_flux_op));
```

Comment:
Perhaps adopt a PETSc-style naming convention on these two functions that do the same thing in different ways.
```suggestion
    PetscCall(CreateCeedInteriorFluxOperator_SingleComm(*config, mesh, &interior_flux_op));
  } else {
    PetscCall(CreateCeedInteriorFluxOperator_DoubleComm(*config, mesh, &interior_flux_op));
```